lib/dao.js

```diff
@@ -2334,6 +2335,24 @@ DataAccessObject.prototype.updateAttributes = function updateAttributes(data, op
     if (err) return cb(err);
     data = ctx.data;
 
+    if (strict) {
+      var props = Model.definition.properties;
+      var keys = Object.keys(data);
+      var result = {};
+      for (var i = 0; i < keys.length; i++) {
+        key = keys[i];
+        if (props[key]) {
+          result[key] = data[key];
+        } else if (strict === 'throw') {
+          cb(new Error('Unknown property: ' + key));
+          return;
+        } else if (strict === 'validate') {
+          inst.__unknownProperties.push(key);
+        } 
+      }
+      data = removeUndefined(result);
+    }
+
```

Comment:
Hmmm... that's tricky... the only way I can think of right now would be to pass the raw data as options to the connector as well, and then let the connector handle it (like `parseUpdateData`).

I must admit, having this raw power is already a lot less safe - this is why it needs to be explicitly set `allowExtendedOperators: true`. So I guess any form of `strict` being enabled wouldn't probably help much anyway.

What would you suggest?